teamup-server/src/main/java/com/project/teamup/security/JwtUserDetailsService.java

```diff
@@ -15,10 +16,15 @@
 public class JwtUserDetailsService implements UserDetailsService {
     @Autowired
     private UserRepository userRepository;
+    @Autowired
+    private LoginAttemptService loginAttemptService;
 
     @Override
     public UserDetails loadUserByUsername(String username) throws UsernameNotFoundException {
         Optional<com.project.teamup.model.User> user = userRepository.findByUsername(username);
+        if (loginAttemptService.isBlocked(username)) {
```

Comment:
Please change the username parameter with the user found above and move your logic in the if statement declared below

Reply:
Fixed in commit e196aa0.